app/routes/surveys/components/SubmissionEditor/SubmissionEditor.js

```diff
@@ -40,6 +40,9 @@ const SubmissionEditor = ({
         Spørreundersøkelse for arrangementet{' '}
         <Link to={`/events/${survey.event.id}`}>{survey.event.title}</Link>
       </div>
+      <div className={styles.surveyTime}>
```

Comment:
What a retarded class name - it only adds color and has very little to do with time ... oh well 🤷🏼